modules/geochemistry/test/tests/interrogate_reactions/tests

```diff
@@ -0,0 +1,112 @@
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
+    expect_out = 'Not enough activites known to compute equilibrium temperature for reaction
+  Gypsum = 2\*H2O \+ 1\*Ca\+\+ \+ 1\*SO4--  .  log10\(K\) = -4.443
+Gypsum.  T = 44.1212'
+    requirement = "The geochemistry interrogator shall be able to compute temperature at equilibrium, given enough information about activity values"
+    issues = '#14981'
+    design = 'eqm_temp_a.md GeochemicalModelInterrogator.md GeochemicalModelRoot.md'
+  [../]
+  [./hematite]
+    type = 'RunApp'
+    input = 'hematite.i'
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
+    expect_out = '\(A_H\+\)\^-4 \(A_O2\(aq\)\)\^0.5 = 10\^3.068
+\(A_H\+\)\^-6 \(A_e-\)\^-2 = 10\^46.07'
+    requirement = "The geochemistry interrogator shall be output equations involving pH and pe for equilibrium reactions"
+    issues = '#14981'
+    design = 'pH_pe.md GeochemicalModelInterrogator.md GeochemicalModelRoot.md'
+  [../]
+  [./muscovite]
+    type = 'RunApp'
+    input = 'muscovite.i'
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
+    expect_out = '\(A_K\+\)\^1 \(A_Al\+\+\+\)\^3 \(A_SiO2\(aq\)\)\^3 \(A_H\+\)\^-10 = 10\^14.56
+\(A_K\+\)\^1 \(A_H\+\)\^-1 = 10\^3.421'
+    requirement = "The geochemistry interrogator shall output activity ratios for different choice of basis components."
+    issues = '#14981'
+    design = 'activity_ratios.md GeochemicalModelInterrogator.md GeochemicalModelRoot.md'
+  [../]
+  [./muscovite2]
+    type = 'RunApp'
+    input = 'muscovite2.i'
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
+    expect_out = '\(A_H2O\)\^6 \(A_K\+\)\^1 \(A_Al\+\+\+\)\^3 \(A_SiO2\(aq\)\)\^3 \(A_H\+\)\^-10 = 10\^14.56
+\(A_H2O\)\^6 \(A_K\+\)\^1 \(A_Al\+\+\+\)\^3 \(A_H\+\)\^-10 = 10\^26.56
+\(A_K\+\)\^-2 \(A_H\+\)\^2 = 10\^-9.681
+\(A_K\+\)\^-2 \(A_SiO2\(aq\)\)\^-6 \(A_H\+\)\^2 = 10\^14.31
+\(A_K\+\)\^-2 \(A_H\+\)\^2 = 10\^-8.686
+\(A_K\+\)\^-2 \(A_SiO2\(aq\)\)\^-6 \(A_H\+\)\^2 = 10\^14.31
+\(A_K\+\)\^-2 \(A_H\+\)\^2 = 10\^-1.967'
+    requirement = "The geochemistry interrogator shall output activity ratios for different choice of basis components."
+    issues = '#14981'
+    design = 'activity_ratios.md GeochemicalModelInterrogator.md GeochemicalModelRoot.md'
+  [../]
+  [./pyrite]
+    type = 'RunApp'
+    input = 'pyrite.i'
+    allow_warnings = true
+    method = 'OPT'
+    threading = '!pthreads'
+    expect_out = 'Pyrite = - 1\*H2O \+ 1\*Fe\+\+ \+ 2\*SO4-- \+ 2\*H\+ - 3.5\*O2\(aq\)  .  log10\(K\) = 217.4
+Pyrite = - 3.5\*H2O \+ 1\*Fe\(OH\)3\(ppd\) \+ 2\*SO4-- \+ 4\*H\+ - 3.75\*O2\(aq\)  .  log10\(K\) = 221
+Pyrite = - 1\*H2O \+ 1\*Fe\+\+ \+ 2\*SO4-- \+ 2\*H\+ - 3.5\*O2\(aq\)  .  log10\(K\) = 217.4
+Pyrite = - 1\*H2O \+ 1\*Fe\+\+ \+ 2\*H2S\(aq\) - 2\*H\+ \+ 0.5\*O2\(aq\)  .  log10\(K\) = -45.39
+Pyrite = - 1\*H2O \+ 1\*Fe\+\+ \+ 1.75\*H2S\(aq\) - 1.5\*H\+ \+ 0.25\*SO4--  .  log10\(K\) = -12.54'
```

Comment:
I think you missed a backslash here 😆 (just joking!)

Reply:
Thank goodness for text editors that can do find-and-replace!